### PR TITLE
Add sorting to product list

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it adds sorting prototype to product list.

### Screenshots
Screenshot 2019-09-13 at 16 47 49

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted to `.pot` file.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/utils/sort.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    sort: field
+  };
+}
+
+export function getOrderDirection(asc: boolean): OrderDirection {
+  return asc ? OrderDirection.ASC : OrderDirection.DESC;
+}
+
+export function getArrowDirection(asc: boolean): TableCellHeaderArrowDirection {
+  return asc ? "asc" : "desc";
```
Comment (benekex2) [inline src/utils/sort.ts]: You should use `OrderDirection enum`

File: src/components/TableCellHeader/TableCellHeader.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+);
+const InteractiveStory: React.FC = () => {
+  const [direction, setDirection] = React.useState<
+    TableCellHeaderArrowDirection
+  >("asc");
+  const [field, setField] = React.useState<Field>("name");
+
+  const handleHeaderClick = (selectedField: Field) => {
+    if (field === selectedField) {
+      setDirection(direction === "asc" ? "desc" : "asc");
```
Comment (benekex2) [inline src/components/TableCellHeader/TableCellHeader.stories.tsx]: You should use `OrderDirection enum`

File: src/components/TableCellHeader/TableCellHeader.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    } else {
+      setField(selectedField);
+      setDirection("asc");
```
Comment (benekex2) [inline src/components/TableCellHeader/TableCellHeader.stories.tsx]: You should use `OrderDirection enum`

File: src/components/TableCellHeader/TableCellHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      <div
+        className={classNames(classes.labelContainer, {
+          [classes.labelContainerCenter]: textAlign === "center",
+          [classes.labelContainerRight]: textAlign === "right"
+        })}
+      >
+        {!!direction && arrowPosition === "left" && (
+          <ArrowSort
+            className={classNames(classes.arrow, classes.arrowLeft, {
+              [classes.arrowUp]: direction === "asc"
```
Comment (benekex2) [inline src/components/TableCellHeader/TableCellHeader.tsx]: You should use `OrderDirection enum`

File: src/components/TableCellHeader/TableCellHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            })}
+          />
+        )}
+        <div className={classes.label}>{children}</div>
+        {!!direction && arrowPosition === "right" && (
+          <ArrowSort
+            className={classNames(classes.arrow, {
+              [classes.arrowUp]: direction === "asc"
```
Comment (benekex2) [inline src/components/TableCellHeader/TableCellHeader.tsx]: You should use `OrderDirection enum`

File: src/utils/sort.ts (hunk 1 above)
Comment (dominik-zeglen) [reply src/utils/sort.ts]: `TableCellHeader` will eventually end up in Macaw, so it's necessary to keep UI and API separated.

File: src/components/TableCellHeader/TableCellHeader.stories.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply src/components/TableCellHeader/TableCellHeader.stories.tsx]: Same as above

**Comment by dominik-zeglen** [comment]
After speaking to @maarcingebala we agreed to keep sorting by name as a default sort field.

**Pull request merged**